src/lib.rs

```diff
@@ -454,15 +435,13 @@ impl System {
                 .recv(&addr.control_rx, |msg| match msg {
                     Ok(control) => Received::Control(control),
                     Err(RecvError::Disconnected) => {
-                        Received::Error(ActorError::ControlChannelDisconnected {
-                            actor_name: A::name(),
-                        })
+                        unreachable!("We keep control_tx alive through addr, should no happen.")
```

Comment:
typo: `should not happen`

Reply:
As per the suggestion from @mcginty on the other PR, let's switch to use `panic!()` here.

Reply:
:+1: